src/main/java/io/cdap/e2e/pages/locators/CdfStudioLocators.java

```diff
@@ -145,4 +148,25 @@ public static WebElement macroButton(String pluginProperty) {
   public static WebElement macroInput(String pluginProperty) {
     return SeleniumDriver.getDriver().findElement(By.xpath("//input[@data-cy='" + pluginProperty + "']"));
   }
+
+  public static WebElement getDataPipelineTypeDropdownOption(String option) {
+    String xpath = "//div[contains(@class, 'left-top-section')]//select//option[contains(@label, '" + option + "')]";
+    return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
+  }
+
+  public static WebElement getPluginNameInList(String pluginName) {
+    String xpath = "//div[contains(@class, 'plugin-name')][normalize-space(text()) = '" + pluginName + "']";
+    return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
+  }
+
+  public static WebElement getPluginNameInDiagramContainer(String pluginName) {
```

Comment:
Where is this used? I think more intuitive name is required here. What is diagram container here? Do you mean pipeline studio canvas? If yes, we can rename it to something like `locatePluginNodeInCanvas(String pluginName)`.

Reply:
Similarly in other method names replace `get` with `locate`

Reply:
Yes, it's for the plugin node in the studio canvas. Updated the locator names as suggested.